### PR TITLE
PVD collection: Consider full path to vtk instead of just the filename

### PR DESCRIPTION
**Pull request by sebastianpech**
Implements #51

### CODE REVIEW

File: src/gridtypes/ParaviewCollection.jl [hunk 1]
```diff
@@ -20,7 +20,7 @@ function collection_add_timestep(pvd::CollectionFile, datfile::VTKFile,
     xroot = root(pvd.xdoc)
     xMBDS = find_element(xroot, "Collection")
     xDataSet = new_child(xMBDS, "DataSet")
-    fname = splitdir(datfile.path)[2]
+    fname = relpath(datfile.path |> abspath ,pvd.path |> abspath |> splitdir |> first)
```
Comment (fredrikekre) [inline src/gridtypes/ParaviewCollection.jl]: Maybe
```suggestion
    fname = relpath(abspath(datfile.path), first(splitdir(abspath(pvd.path))))
```
to follow the style of the rest of the code? :)

Comment (jipolanco) [reply src/gridtypes/ParaviewCollection.jl]: Perfect, thanks!

Comment (sebastianpech) [reply src/gridtypes/ParaviewCollection.jl]: Also fine. I just like the clean look you get when using `|>`

**Comment by jipolanco** [comment]
Thanks for the PR!

File: src/gridtypes/ParaviewCollection.jl (hunk 1 above)
Comment (fredrikekre) [reply src/gridtypes/ParaviewCollection.jl]: > I just like the clean look you get when using |>

Well, I like the clean look when you don't use it :)

**Pull request merged**